toxcore/tox_private.h

```diff
@@ -152,8 +152,8 @@ typedef enum Tox_Err_Dht_Get_Nodes {
  *
  * @return true on success.
  */
-bool tox_dht_get_nodes(const Tox *tox, const uint8_t *public_key, const char *ip, uint16_t port,
-                       const uint8_t *target_public_key, Tox_Err_Dht_Get_Nodes *error);
+bool tox_dht_nodes_request(const Tox *tox, const uint8_t *public_key, const char *ip, uint16_t port,
```

Comment:
Should we call it "tox_dht_send_nodes_request"?

Reply:
I guess we should, while we are at it.

Reply:
https://github.com/TokTok/c-toxcore/pull/2860/commits/a60fb7375bbce31bfe1f26a52f15ca830d5f7a92

Reply:
LGTM.